openedx/adg/lms/applications/helpers.py

```diff
@@ -0,0 +1,21 @@
+"""
+Helper methods for applications
+"""
+from django.core.validators import ValidationError
+from django.utils.translation import ugettext as _
+
+from .constants import LOGO_IMAGE_MAX_SIZE
+
+
+def validate_logo_size(file_):
+    """
+    Validate maximum allowed file upload size, raise validation error if file size exceeds.
+
+    Arguments:
+         file_(object): file that needs to be validated for size
+    """
+    size = getattr(file_, 'size', None)
+    if size and LOGO_IMAGE_MAX_SIZE < size:
+        raise ValidationError(
+            _('File size must not exceed {size} kB').format(size=LOGO_IMAGE_MAX_SIZE / 1024)
```

Comment:
Please place them in one line as it doesn't required to make it to next line as it ll have 110 characters but our limit is 120 characters per line